tools/niminst/niminst.nim

```diff
@@ -505,12 +505,20 @@ proc writeInstallScripts(c: var ConfigData) =
     writeFile(deinstallShFile, generateDeinstallScript(c), "\10")
     inclFilePermissions(deinstallShFile, {fpUserExec, fpGroupExec, fpOthersExec})
 
+template gatherFiles(fun, libpath, outDir) =
+  block:
```

Comment:
Nit picking: I don't like ``block`` inside templates, Nim has hygenic macros.

Reply:
they're hygienic but template defs get exposed:
```nim
template copySrc() = echo "ok1"
template gatherFiles() =
  template copySrc() = echo "ok2"
gatherFiles()
copySrc() # prints ok2
```

Reply:
1. It doesn't matter for this small tool.
2. You can use a ``{.gensym.}`` annotation to prevent that.

Reply:
ok {.gensym.} is good